pkg/parsers/parsers.go

```diff
@@ -31,7 +32,7 @@ func ParseKeyValueOpt(opt string) (string, string, error) {
 //     03,1-3      <- this is gonna get parsed as [1,2,3]
 //     3,2,1
 //     0-2,3,1
-func ParseUintList(val string) (map[int]bool, error) {
+func ParseUintList(val string, maximum int) (map[int]bool, error) {
```

Comment:
Should we keep the signature (for not breaking downstream-dependencies) and add a new method `ParsUintListMaximum` or something ?

Reply:
Ah, yes; I could add a new one.